aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -469,6 +469,9 @@ Resources:
 <%  unless load_balancer -%>
           - !ImportValue VPC-ELBSecurityGroup # Accept HTTP traffic on daemon
 <%  end -%>
+<%  unless rack_env?(:adhoc) -%>
+    DisableApiTermination: true
```

Comment:
Given that ERB and YAML nesting don’t interact well, a single line might be more readable. Something like:
```DisableApiTermination: <%= rack_env?(:adhoc) ? false : true %>```